Production_scripts/Process_extracts/Produce_extract_tests/extract_ae_tests.R

```diff
@@ -0,0 +1,45 @@
+####################################################
+# A&E Extract Tests
+# Original Authors - Catherine Holland
+# Original Date - May 2022
+# Written/run on - RStudio Server
+# Version of R - 3.6.1
+# Description - Produce tests for source linkage files:
+#               A&E processed file.
+#####################################################
+
+# Load packages
+library(createslf)
+library(openxlsx)
+library(slfhelper)
+
+
+# Read in Data-----------------------------------------
+
+year <- "1920"
+
+# Read new data file
+new_data <- readr::read_rds(get_source_extract_path(year, "AE", ext = "rds"))
+# zsav until rds file avaliable
+new_data <- haven::read_sav(get_source_extract_path(year, "AE", ext = "zsav")) %>%
+  rename(postcode = "Postcode")
+
+# Read current SLF episode file and filter for 01B and GLS records
+existing_data <- get_existing_data_for_tests(new_data = new_data)
+
+
+# Produce comparison-------------------------------------
+# Compare new file with existing slf data
+comparison <- produce_test_comparison(
+  produce_source_acute_tests(existing_data),
+  produce_source_acute_tests(new_data)
+)
```

Comment:
If we are using this function for others as well we can re name this to something more generic